uvicorn/supervisors/basereload.py

```diff
@@ -64,7 +64,6 @@ def startup(self) -> None:
         self.process.start()
 
     def restart(self) -> None:
-        self.mtimes: Dict[str, float] = {}
```

Comment:
why this is dropped ?

Reply:
It was moved to StartReload as it is not used anywhere else